### ISSUE TITLE
LIgov / IDme LOA: Update content of non-LOA3 alert and don't show to MHV-acct users

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-team#83069]
### User story

As a Veteran signed in using a non-LOA3 **Login.gov** or **ID.me** credential, I want the My HealtheVet landing page to tell me what steps I need to take to verify my credential and register with My HealtheVet, so that I can get access to my health data and tools with as little frustration as possible.

As a Veteran signed in using a non-LOA3 **My HealtheVet basic** credential, I no longer want to see this alert, because it is not correctly telling me what steps I need to take. (implementing a correct alert for this credential is covered by separate ticket #77357)

### Notes
- Design and flow in [Figma](https://www.figma.com/design/CAChU51fWYMZsgDR5RXeSc/MHV-Landing-Page?node-id=942-11704&t=sPISB9TZDaZX70AS-4)
- A similar version of this alert for My HealtheVet basic (non-LOA3) credentials [covered by this separate ticket](https://app.zenhub.com/workspaces/mhv-on-vagov-landing-page-62619a987d74510018ecc546/issues/gh/department-of-veterans-affairs/va.gov-team/77357)

**Screen shot**
<img width="906" alt="Screenshot 2024-07-01 at 3 11 31 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/115033532/e27155df-fa76-4174-845c-b65ee9383c82">
<img width="385" alt="Screenshot 2024-07-01 at 3 19 30 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/115033532/1f678b4a-2b1b-44e2-84a2-d8ae0e9b2b10">

### Alert content
`H2`
Verify and register your account to access My HealtheVet

`Body text`
We need you to verify your identity and register your account before you can access My HealtheVet. These steps help us keep your health information safe and prevent fraud and identity theft. 
1. Verify your identity for your {%account type%} account. This one-time process often takes about 10 minutes. You’ll need to provide certain personal information and identification.
2. Then we’ll ask you to register with My HealtheVet by confirming your personal information and relationship with VA health care. You’ll need to register before you can access your messages, medications, and medical records. 

`primary action link`
Verify your identity with {%account provider%} [URL: va.gov/verify]

## Possible tasks:
- [x] Update the alert content to match image above - **or see [Figma file](https://www.figma.com/design/CAChU51fWYMZsgDR5RXeSc/MHV-Landing-Page?node-id=3092-3800&t=XrougVpPBRlIjdFY-4)**
- [x] Ensure the alert content conditionally pulls the credential name (ID.me or Login.gov), and no longer displays for My HealtheVet basic credentials
- [x] Update the landing page design surrounding the alert to match [Figma file here](https://www.figma.com/design/CAChU51fWYMZsgDR5RXeSc/MHV-Landing-Page?node-id=602-760&t=Hcauqz45dpAetAOr-4) - should not see secondary navigation or any of the gray boxes for the health tools

## Acceptance criteria
- [x] **Created feature branch so that this ticket and #77357 are released together**
- [x] Implemented / modified alert for non-LOA3 Login.gov and ID.me credentials only
- [x] Alert displays with the simplified landing page design depicted in the Figma (linked above)
- [ ] UX review
- [ ] QA review
- [ ] A11y review

**Comment by radavis** [comment]
As we're probably going to continue adding alerts to MHV Home, I started a [document](https://github.com/department-of-veterans-affairs/va.gov-team/blob/063475cef825973db5dc88ed2271d69063a98ba1/products/health-care/digital-health-modernization/engineering/mhv-home-alerts.md) on how we could better organize these.

**Comment by jzucadi** [comment]
This work is blocked due to more UX changes being decided on, details in this thread: https://dsva.slack.com/archives/C0581MN69TJ/p1722261242087759?thread_ts=1722260409.416559&cid=C0581MN69TJ

**Comment by allanto-ah** [comment]
Environment: Staging

Verified that this alert is showing as expected.


<img width="1283" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzFQQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c6b31b343795fa5db43261590b6be2e463b3d6dc/Screenshot%202024-08-09%20at%2011.33.43%E2%80%AFAM.png" alt="Screenshot 2024-08-09 at 11.33.43 AM.png" />

**Comment by wesrowe** [comment]
@allanto-ah, have you recorded your test cases somewhere? This ticket is a little complicated, in terms of how it fits in with other functionality. Note that there was dependency between this ticket and #77357, where both needed to be merged at the same time – otherwise we could leave some users without an alert at all. It would make sense to me for them to be tested together.

@jzucadi, can you confirm that these two stories were merged together? (using a feature branch, I believe.)

cc @LynneJohnson224

**Comment by allanto-ah** [comment]
Sorry for the delayed response to this ticket. Yeah they were merged together. For 77357, found out that the MHV login doesn't work in staging, so we couldn't test to get that alert to show up. Instead I created a MHV account in production to test

Test cases: 
1. Verify unregistered users do not see the verify and register alert
2. Verify unverified users do see the verify and register alert
3. Verify registered users do not see any alerts
4. Verify cerner users do not see any alerts

Regression:
1. Verify secondary nav appears as expected for users that should see it
